cnf/ext/central.mvn

```diff
@@ -197,4 +197,7 @@ org.apache.zookeeper:zookeeper:3.4.13
 org.apache.aries.component-dsl:org.apache.aries.component-dsl.component-dsl:1.2.2
 org.apache.aries.typedevent:org.apache.aries.typedevent.bus:0.0.1-SNAPSHOT
 
-
+org.apache.felix:org.apache.felix.feature:0.0.1-SNAPSHOT
```

Comment:
Isn't there already an RC1 version in maven central? (RC2 is under ballot.)

Reply:
i use this one 
https://repository.apache.org/content/groups/snapshots/org/apache/felix/org.apache.felix.feature/0.9.1-SNAPSHOT/